packages/features/bookings/lib/handleNewBooking.ts

```diff
@@ -1627,6 +1628,24 @@ async function handler(
       };
     }
   }
+  if (
+    Object.prototype.hasOwnProperty.call(eventType, "bookingLimits") ||
+    Object.prototype.hasOwnProperty.call(eventType, "durationLimits")
+  ) {
+    const startAsDate = dayjs(reqBody.start).toDate();
+    if (eventType.bookingLimits) {
+      await checkBookingLimits(
+        eventType.bookingLimits as IntervalLimit,
+        startAsDate,
+        eventType.id,
+        eventType.schedule?.timeZone
```

Comment:
type error here

Reply:
A new fourth parameter got added for checkBookingLimits() last month on the main branch.
It is complaining about that.
```suggestion
        rescheduleUid,
        eventType.schedule?.timeZone
```